.github/workflows/prow-pr-automerge.yml

```diff
@@ -0,0 +1,17 @@
+# This Github workflow will check every hour for PRs with the lgtm label and will attempt to automatically merge them.
+# If the hold label is present, it will block automatic merging.
+
+name: "Prow merge on lgtm label"
```

Comment:
will this merge PRs with lgtm label? can we set it up to merge only if both lgtm and approve is on the PR?

Reply:
approve is needed for merging